controllers/httproute_controller.go

```diff
@@ -254,6 +254,30 @@ func (r *HTTPRouteReconciler) reconcileHTTPRouteResource(ctx context.Context, ht
 		r.eventRecorder.Event(httpRoute.K8sObject(), corev1.EventTypeWarning, k8s.HTTPRouteventReasonFailedAddFinalizer, fmt.Sprintf("Failed add finalizer due to %v", err))
 	}
 
+	ok := r.areBackendRefsIpFamiliesValid(ctx, httpRoute.Inner())
+
+	if !ok {
+		httpRouteOld := httpRoute.Inner().DeepCopy()
+
+		setupRouteStatusParents(httpRoute.Inner())
+
+		updateRouteCondition(httpRoute, metav1.Condition{
+			Type:               string(gateway_api.RouteConditionAccepted),
+			Status:             metav1.ConditionFalse,
+			ObservedGeneration: httpRoute.Inner().Generation,
+			Reason:             string(gateway_api.RouteReasonUnsupportedValue),
+			Message:            fmt.Sprintf("Dual stack Service is not supported"),
+		})
+
+		if err := r.client.Status().Patch(ctx, httpRoute.Inner(), client.MergeFrom(httpRouteOld)); err != nil {
+			glog.V(2).Infof("updateHTTPRouteStatus: Patch() received err %v \n", err)
+
+			return errors.Wrapf(err, "failed to update httproute status")
+		}
+
+		return errors.New("Dual stack Service is not supported")
```

Comment:
can you add to error message what kind of dual stack it is

Reply:
> what kind of dual stack it is

I don't understand what do you mean by that?

Reply:
for example you have this error in another place, which looks nicer and more informative
`errors.New("Invalid IpFamilies, Lattice Target Group doesn't support dual stack ip addresses")`

Reply:
i see, I'll change the error message to be the same as that.